lua-tree/share/lua/5.1/kong/tools/utils.lua

```diff
@@ -1267,7 +1267,7 @@ _M.yield = function() return end
 local try_decode_base64
 do
   local decode_base64    = ngx.decode_base64
-  local decode_base64url = require "ngx.base64".decode_base64url
+  local decode_base64url = ngx.decode_base64url
```

Comment:
Addresses:
```
lua-tree/share/lua/5.1/kong/tools/utils.lua:1270: module ngx.base64 not found:
    no field package.preload['ngx.base64']
    open /ngx/base64.lua: file does not exist
    open lua-tree/share/lua/5.1/ngx/base64.lua: file does not exist
    open lua-tree/share/lua/5.1/ngx/base64/init.lua: file does not exist,
```

Reply:
there's already `lualibs/go/ngx/base64.go`, which implements the `ngx.decode_base64` function.  i guess the URL variant should be added there, so line 1270 could go unmodified.

Reply:
Adding a `decode_base64url` aliased function to `lualibs/go/ngx/base64.go` was my first approach. However, I noticed that updating the import also resolved the issue. My guess is there may be a runtime error for calls to `ngx.decode_base64url`? I'll look at extending `lualibs/go/ngx/base64.go` and adding some test coverage if I find gaps.

Reply:
> there's already lualibs/go/ngx/base64.go, which implements the ngx.decode_base64 function. i guess the URL variant should be added there, so line 1270 could go unmodified.

We have `encode_base64`, no decode functions. Any I way I change this will be internally inconsistent. I'd like to leave this change as-is without adding any new methods to the go.ngx package.